app/views/comments/new.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'form', comment: @comment %>
```

Comment:
- [ ] Good job on adding comment view form, however, although comments can be added via the post#show, an error still surfaces in an attempt to create a new form via the  `/users/1/posts/1/comments/new` route, like indicated in the previous review. Let's ensure to improve upon this aspect.

**Error in displaying comment view via users/1/posts/1/comments/new:**
![image](https://github.com/eamador1/blog-app/assets/12745474/cc5cec9c-d82a-4a9e-b5b8-45b582b7460b)